lib/SILOptimizer/SILCombiner/SILCombinerMiscVisitors.cpp

```diff
@@ -585,6 +586,80 @@ SILInstruction *SILCombiner::optimizeLoadFromStringLiteral(LoadInst *LI) {
   return Builder.createIntegerLiteral(LI->getLoc(), LI->getType(), str[index]);
 }
 
+SILInstruction *SILCombiner::visitStoreInst(StoreInst *si) {
+  // These optimizations assume that lowered addresses are not enabled.
```

Comment:
"are enabled" I think? "lowered addresses" means opaque values are not in use (or already lowered away)

Reply:
I fixed it.